Altis_Life.Altis/stringtable.xml

```diff
@@ -1490,6 +1490,30 @@
             <Polish>You have sent a message to all EMS Units.</Polish>
 			<Chinesesimp>你已经向所有医护人员发送了消息.</Chinesesimp>
         </Key>
+        <Key ID="STR_CELLMSG_ToPolice">
+            <Original>You have sent a message to all Police Units.</Original>
+            <Czech>You have sent a message to all Police Units.</Czech>
+            <Spanish>You have sent a message to all Police Units.</Spanish>
+            <Russian>You have sent a message to all Police Units.</Russian>
+            <German>Du hast eine Nachricht an alle Polizisten gesendet.</German>
+            <French>Vous avez envoyé un message à tous les Police.</French>
+            <Italian>You have sent a message to all Police Units.</Italian>
+            <Portuguese>You have sent a message to all Police Units.</Portuguese>
+            <Polish>You have sent a message to all Police Units.</Polish>
+			<Chinesesimp></Chinesesimp>
+        </Key>
+        <Key ID="STR_CELLMSG_ToAdmin">
+            <Original>You have sent a message to all Admins.</Original>
+            <Czech>You have sent a message to all Admins.</Czech>
+            <Spanish>You have sent a message to all Admins.</Spanish>
+            <Russian>You have sent a message to all Admins.</Russian>
+            <German>Du hast eine Nachricht an alle Admins gesendet.</German>
+            <French>Vous avez envoyé un message à tous les Admins.</French>
+            <Italian>You have sent a message to all Admins.</Italian>
+            <Portuguese>You have sent a message to all Admins.</Portuguese>
+            <Polish>You have sent a message to all Admins.</Polish>
+			<Chinesesimp></Chinesesimp>
```

Comment:
```suggestion
            <Chinesesimp></Chinesesimp>
```

Reply:
(and the others)

Reply:
which others? looks all normal to me

![atom_ZAe7L17oJw](https://user-images.githubusercontent.com/13716626/77072207-a08d4580-69ed-11ea-80e5-c70bf28c72a8.png)